src/docs/sdk/event-payloads/message.mdx

```diff
@@ -25,8 +25,8 @@ help to group similar messages into the same issue.
 
 `params`
 
-: _Optional_: A list of formatting parameters, preferably strings. Non-strings
-  will be coerced to strings.
+: _Optional_. A list or dictionary of formatting parameters, preferably strings.
+  Non-strings will be coerced to strings.
```

Comment:
On Mobile SDKs, it's a `List<String>` only (strongly typed).
Relay spec is `any`, so it can be anything.